addons/website_slides_survey/static/src/xml/website_slide_upload.xml

```diff
@@ -3,33 +3,31 @@
     <t t-name="website.slide.upload.modal.certification">
         <div>
             <form class="clearfix">
-                <div class="form-group row">
-                    <label for="certification_id" class="col-form-label col-md-3">Certification</label>
-                    <div class="controls col-md-9">
-                        <input class="form-control" id="certification_id" required="required"/>
-                    </div>
+                <div id="warning-no-certif" class="o_hidden" style="color: red;text-align: right;">
+                    There is no certification available, please contact your administrator.
                 </div>
-                <t t-call="website.slide.upload.modal.common"/>
-                <div class="form-group row">
-                    <div class="col-md-4">
+                <div class="row">
+                    <div id="o_wslides_js_slide_upload_left_column" class="col">
+                        <div class="form-group">
+                            <label for="certification_id" class="col-form-label col-md-3">Certification</label>
+                            <input class="form-control" id="certification_id" required="required"/>
+                        </div>
+                        <div class="form-group">
+                            <label for="upload" class="col-form-label">Choose an Image</label>
+                            <input id="uploadCertification" name="file" class="form-control" accept="image/*" type="file" required="required"/>
+                        </div>
+                        <t t-call="website.slide.upload.modal.common"/>
+                        <canvas id="data_canvas" class="d-none"></canvas>
+                    </div>
+ 
+                    <div id="o_wslides_js_slide_upload_preview_column" class="col-md-6 d-none">
```

Comment:
are `o_wslides_js_slide_upload_left_column` and `o_wslides_js_slide_upload_preview_column` ids necessary ?